src/django_components/urls.py

```diff
@@ -0,0 +1,5 @@
+from django.urls import include, path
+
+urlpatterns = [
+    path("components/", include("django_components.dependencies")),
```

Comment:
I didn't mention it explicitly in other comments, but with this change, if users want to use JS or CSS with the components (whether as inlined JS / CSS or inside the `Media` class, they will MUST add our URL router to theirs.